### PR TITLE
add all functional for MyArrayList

### PR DESCRIPTION
**Pull request by Illia1927**

### CODE REVIEW

File: src/mate/academy/myArrayList/MyArrayList.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    private Object[] array;
+    private int size;
+
+    public MyArrayList() {
+        array = new Object[10];
+    }
+
+    public MyArrayList(int capacity) {
+        if (capacity < 0)
+            throw new IllegalArgumentException("You was inputed incorect data.");
```
Comment (Siril-O) [inline src/mate/academy/myArrayList/MyArrayList.java]: Add more formal message, like' invalid capacity '

File: src/mate/academy/myArrayList/MyArrayList.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return size == 0;
+    }
+
+    @Override
+    public boolean contains(Object value) {
+        if (value == null) {
```
Comment (Siril-O) [inline src/mate/academy/myArrayList/MyArrayList.java:36]: this is not correct your array can contain multiple null elements and your array does not distinguish between added by user null elements and empty array indexes, do not allow add null elements

File: src/mate/academy/myArrayList/MyArrayList.java [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+            array[index + i] = someArray[i];
+        }
+        size--;
+        array[size] = null;
+        return buf;
+    }
+
+    @Override
+    public boolean remove(Object o) {
+        return false;
```
Comment (Siril-O) [inline src/mate/academy/myArrayList/MyArrayList.java:123]: Why not implemented, if you don`t  want this extra methods, create your interface . with methods you like

**Review by Siril-O — CHANGES_REQUESTED** [review]

File: src/mate/academy/myArrayList/MyArrayList.java (hunk 2 above)
Comment (Illia1927) [reply src/mate/academy/myArrayList/MyArrayList.java:36]: @Siril-O i don't understand you, what i should change in this method?

File: src/mate/academy/myArrayList/MyArrayList.java (hunk 3 above)
Comment (Illia1927) [reply src/mate/academy/myArrayList/MyArrayList.java:123]: @Siril-O In my implementation, I made the "remove" method by index. This is not enough?

**Pull request merged**